SharyLane/src/test/java/SignUpTest.java

```diff
@@ -0,0 +1,120 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SignUpTest {
+    WebDriver driver;
+
+    @BeforeMethod
+    public void SetUp() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+    }
+
+    private static final String baseUrl = "https://www.sharelane.com/cgi-bin/register.py";
+
+    private void SendZipCode(String zipCode) {
+        //Open url code zipcode page
+        driver.get(baseUrl);
+        //Input 5 digits into input field
+        driver.findElement(By.name("zip_code")).sendKeys(zipCode);
+        //click continue
+        driver.findElement(By.cssSelector("[value=Continue]")).click();
+    }
+
+    private void Registration(String pass, String confirmPass) {
+        driver.findElement(By.name("first_name")).sendKeys("stef");
+        driver.findElement(By.name("last_name")).sendKeys("lilk");
+        driver.findElement(By.name("email")).sendKeys("stefl1234@mail.ru");
+        driver.findElement(By.name("password1")).sendKeys(pass);
+        driver.findElement(By.name("password2")).sendKeys(confirmPass);
+        driver.findElement(By.cssSelector("[value=Register]")).click();
+    }
+
+    @Test
+    public void FiveDigitsZipCodeTest() {
```

Comment:
Please follow Java naming convention https://comaqa.gitbook.io/java-automation/stil-napisaniya-koda/konvenciya-imenovaniya#:~:text=%D0%9A%D0%BE%D0%BD%D0%B2%D0%B5%D0%BD%D1%86%D0%B8%D1%8F%20%D0%B8%D0%BC%D0%B5%D0%BD%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%20%2D%20Java%20%D0%B4%D0%BB%D1%8F%20%D0%B0%D0%B2%D1%82%D0%BE%D0%BC%D0%B0%D1%82%D0%B8%D0%B7%D0%B0%D1%82%D0%BE%D1%80%D0%BE%D0%B2&text=%D0%9A%D0%BE%D0%BD%D0%B2%D0%B5%D0%BD%D1%86%D0%B8%D1%8F%20%D0%B8%D0%BC%D0%B5%D0%BD%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%20%D0%B4%D0%B5%D0%BB%D0%B0%D0%B5%D1%82%20%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D1%83%20%D0%B1%D0%BE%D0%BB%D0%B5%D0%B5,%D0%B1%D1%8B%D1%82%D1%8C%20%D0%BF%D0%BE%D0%BB%D0%B5%D0%B7%D0%BD%D1%8B%D0%BC%20%D0%B2%20%D0%BF%D0%BE%D0%BD%D0%B8%D0%BC%D0%B0%D0%BD%D0%B8%D0%B8%20%D0%BA%D0%BE%D0%B4%D0%B0.